src/utils/write.ts

```diff
@@ -31,10 +41,13 @@ export const replaceSelection = async (
  * Update a file based on the FileChange information
  * @param fileChange Contains the information about the content to be changed
```

Comment:
We could improve the JSDoc of this method by adding a description for the second parameter `vscode` here.